docs/docs/recipes.md

```diff
@@ -218,6 +218,220 @@ export default NonPageComponent
 - [More on the useStaticQuery hook](/docs/use-static-query/)
 - [Visualize your data with GraphiQL](/docs/introducing-graphiql/)
 
+### Limiting
+
+The number of results returned from a query can be limited to a number you specify. Helpful if you only need a few pieces of data or need to [paginate data](/docs/adding-pagination/).
+
+#### Prerequisites
+
+- A Gatsby site with [nodes in the GraphQL data layer](/docs/node-interface/) (all sites have some nodes like `allSitePage` and `sitePage` created automatically)
+
+#### Directions
+
+1. Run `gatsby develop` to start the development server.
+2. Open a tab in your browser at: `http://localhost:8000/___graphql`.
+3. Add a query in the editor with the following fields on `allSitePage` to start off:
+
+```graphql
+{
+  allSitePage {
+    edges {
+      node {
+        id
+        path
+      }
+    }
+  }
+}
+```
+
+4. Add a `limit` argument to the `allSitePage` field and give it an integer value `3`.
+
+```graphql
+{
+  allSitePage(limit: 3) { // highlight-line
+    edges {
+      node {
+        id
+        path
+      }
+    }
+  }
+}
+```
+
+5. Click the play button in the GraphiQL page and the data in the `edges` field will be limited to the number specified.
+
+#### Additional resources
+
+- [Gatsby GraphQL reference for limiting](/docs/graphql-reference/#limit)
+- Live example:
+
+<iframe title="Limiting returned data" src="https://711808k40x.sse.codesandbox.io/___graphql?query=%7B%0A%20%20allSitePage(limit%3A%203)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20path%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&explorerIsOpen=false" width="600" height="300"></iframe>
+
+### Sorting
+
+The ordering of your results can be specified with the `sort` argument. You can specify which fields to sort by and the order to sort in.
+
+#### Prerequisites
+
+- A Gatsby site with [nodes in the GraphQL data layer](/docs/node-interface/) (all sites have some nodes like `allSitePage` and `sitePage` created automatically)
+- Queryable fields that are prefixed with 'all' (individual nodes that are queried don't need to be sorted)
+
+#### Directions
+
+1. Run `gatsby develop` to start the development server.
+2. Open a tab in your browser at: `http://localhost:8000/___graphql`
+3. Add a query in the editor with the following fields on `allSitePage` to start off:
+
+```graphql
+{
+  allSitePage {
+    edges {
+      node {
+        id
+        path
+      }
+    }
+  }
+}
+```
+
+4. Add a `sort` argument to the `allSitePage` field and give it an object with the a `fields` and `order` attribute. The value for `fields` can be a field or an array of fields to sort by (this example uses the `path` field), the `order` can be either `ASC` or `DESC` for ascending and descending respectively.
```

Comment:
Should we mention that `order` can also be an array -- i.e. when sorting by an array of fields like `fields: [first, second]`, it is possible to sort by `first` in ascending order, and `second` in descending order with `order: [ASC, DESC]`

Reply:
I do like that idea, I battled with including a lot of more details (and even including as much detail as I did) with the format of the recipes though since they're meant to be short. Is it better to just limit to one really simple use case or spell out some more complicated ones too. I aired more on the side of the latter for this one.

Reply:
Yes - i'd say let's keep this simple to start and we can add more detail in the GraphQL docs guides.